src/code42cli/cmds/alerts.py

```diff
@@ -177,13 +178,12 @@ def _call_extractor(
     extractor = _get_alert_extractor(cli_state.sdk, handlers)
     extractor.use_or_query = or_query
     if advanced_query:
-        extractor.extract_advanced(advanced_query)
-    else:
-        if begin or end:
-            cli_state.search_filters.append(
-                ext.create_time_range_filter(f.DateObserved, begin, end)
-            )
-        extractor.extract(*cli_state.search_filters)
+        cli_state.search_filters = advanced_query
+    if begin or end:
```

Comment:
I would expect to see some sort of comparable/equivalent change made for file events for this, am I missing it somewhere?

Reply:
The change for file events is in the `_call_extractor` function, which is used by both `search` and `send-to`.